modules/conversion-center/assets/scss/widgets/link-in-bio-base.scss

```
@@ -151,13 +162,34 @@
 			object-fit: cover;
 		}
 
+		.e-link-in-bio__identity-image-cover {
+			align-self: start;
+			border-color: var(--e-link-in-bio-identity-image-cover-border-color);
```

A general comment: it would be great if we can reduce the length of the variables, without losing meaning. I am thinking `e-link-in-bio` can be reduced to `eb`.

```suggestion
			border-color: var(--eb-identity-image-cover-border-color);
```

---

@hein-obox thanks for flagging this.
Given this naming convention is consistent across a number of variations already merged and requires a wide ranging refactor I would suggest this could be better handled as a separate task. If the consensus is then to reduce the length of variables we should probably consider how we apply naming. A prefix like `--eb-` may be slightly too generic and run the risk of conflict down the line. Perhaps `--e-lib-` ?

---

@willhowat That's perfect. It might be good to discuss it with Louis next week.

---

@louiswol94  just pinging you here so it's on your radar as well